File: packages/ios/Sources/Components/Backdrop/Backdrop.figma.swift
import Figma
import SwiftUI

struct KozmosBackdropConnect: FigmaConnect {
    let component = KozmosBackdrop.self
    let figmaNodeUrl = "https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=613-4791"

    @FigmaEnum(
        "Visibility",
        mapping: [
            "Visible": true,
            "Hidden": false
        ]
    )
    var visible: Bool = true

    var body: some View {
        KozmosBackdrop(visible: self.visible)
    }
}

File: packages/android/src/main/java/com/kozmos/components/Backdrop/Backdrop.figma.kt
package com.kozmos.components.backdrop

import androidx.compose.runtime.Composable
import com.figma.code.connect.Figma
import com.figma.code.connect.FigmaConnect
import com.figma.code.connect.FigmaProperty
import com.figma.code.connect.FigmaType

@FigmaConnect("https://figma.com/design/Yj4O8p6Y9h2Sa9zJVoAiVY?node-id=613-4791")
class KozmosBackdropConnect {
    @FigmaProperty(FigmaType.Enum, "Visibility")
    val visible: Boolean = Figma.mapping(
        "Visible" to true,
        "Hidden" to false
    )

    @Composable
    fun ComponentExample() {
        KozmosBackdrop(visible = visible)
    }
}
